_posts/2020-10-06-bazel-3-6.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: posts
+title: "Bazel 3.6"
+authors:
+  - laurentlb
+---
+
+[Bazel 3.6](https://github.com/bazelbuild/bazel/releases/tag/3.6.0) is a [minor
+release](https://docs.bazel.build/versions/3.5.0/backward-compatibility.html).
+It is an incremental update to Bazel 3.5.
+
+## Incompatible changes
+
+*   The prelude file (`//tools/build_rules:prelude_bazel`) is now processed as
+    a Starlark module, rather than being sourced into the BUILD file textually.
+    This may cause slight breakages depending on the content of the prelude
+    file. For more details, see the discussion in [issue
+    #11940](https://github.com/bazelbuild/bazel/issues/11940).
+*   The flag `--experimental_ui_limit_console_output` is removed. If you
+    use `--experimental_ui_limit_console_output=1` to silence terminal
+    output, you should use `--ui_event_filters=` instead.
+*   The flag `--proto:instantiation_stack` must be enabled with
+    `--record_rule_instantiation_callstack` to see call stack in proto output
+    from Bazel query.
+*   A few other flags are removed:
+    `--experimental_ignore_deprecated_instrumentation_spec`,
```

Comment:
Sub bulleted list or add "and" before the last item.

Reply:
Done